src/analytics/Properties.tsx

```diff
@@ -1070,6 +1070,11 @@ interface WalletConnectProperties {
   [WalletConnectEvents.wc_request_deny_error]: WalletConnectRequestDenyProperties & {
     error: string
   }
+
+  [WalletConnectEvents.wc_copy_request_payload]: {
```

Comment:
I suggest we reuse `WalletConnectDefaultProperties` to follow the other events.

Reply:
haha 🙈 i avoided doing this in the first place because it is hard but i added it now - we were previously using a saga to get all the default props so i abstracted what i could from there to share with this component. i also updated the name of the corresponding dappkit event. since there were non trivial changes maybe you want to take another look @jeanregisser ?

Reply:
great thanks, just did it.